examples/events_sse_transport/main.go

```diff
@@ -18,11 +18,9 @@ package main
 
 import (
 	"flag"
-	"time"
-
 	marathon "github.com/gambol99/go-marathon"
-
-	"github.com/golang/glog"
+	"log"
+	"time"
```

Comment:
Can you please re-order imports everywhere according to https://github.com/golang/go/wiki/CodeReviewComments#imports